src/sentry/static/sentry/app/views/performance/transactionSummary/vitalsChart.tsx

```diff
@@ -0,0 +1,254 @@
+      selection[metric] = false;
+      return selection;
+    }, {});
+
+    const start = this.props.start
+      ? getUtcToLocalDateObject(this.props.start)
+      : undefined;
+
+    const end = this.props.end ? getUtcToLocalDateObject(this.props.end) : undefined;
+    const utc = decodeScalar(router.location.query.utc);
+
+    const legend = {
+      right: 10,
+      top: 0,
+      icon: 'circle',
+      itemHeight: 8,
+      itemWidth: 8,
+      itemGap: 12,
+      align: 'left',
+      textStyle: {
+        verticalAlign: 'top',
+        fontSize: 11,
+        fontFamily: 'Rubik',
+      },
+      selected: seriesSelection,
+    };
+
+    const datetimeSelection = {
+      start: start || null,
+      end: end || null,
+      period: statsPeriod,
+    };
+
+    const chartOptions = {
+      grid: {
+        left: '10px',
+        right: '10px',
+        top: '40px',
+        bottom: '0px',
+      },
+      seriesOptions: {
+        showSymbol: false,
+      },
+      tooltip: {
+        trigger: 'axis',
+        valueFormatter: tooltipFormatter,
+      },
+      yAxis: {
+        axisLabel: {
+          color: theme.gray400,
+          // p75(measurements.fcp) coerces the axis to be time based
+          formatter: (value: number) =>
+            axisLabelFormatter(value, 'p75(measurements.fcp)'),
+        },
+      },
+    };
+
+    return (
+      <React.Fragment>
+        <HeaderTitleLegend>
+          {t('Web Vitals Breakdown')}
+          <QuestionTooltip
+            size="sm"
+            position="top"
+            title={t(
+              `Web Vitals Breakdown reflects the 75th percentile of web vitals over time.`
+            )}
+          />
+        </HeaderTitleLegend>
+        <ChartZoom
+          router={router}
+          period={statsPeriod}
+          projects={project}
+          environments={environment}
+        >
+          {zoomRenderProps => (
+            <EventsRequest
+              api={api}
+              organization={organization}
+              period={statsPeriod}
+              project={[...project]}
+              environment={[...environment]}
+              start={start}
+              end={end}
+              interval={getInterval(datetimeSelection, true)}
+              showLoading={false}
+              query={query}
+              includePrevious={false}
+              yAxis={YAXIS_VALUES}
+            >
+              {({results, errored, loading, reloading}) => {
+                if (errored) {
+                  return (
+                    <ErrorPanel>
+                      <IconWarning color="gray500" size="lg" />
+                    </ErrorPanel>
+                  );
+                }
+                const colors =
+                  (results && theme.charts.getColorPalette(results.length - 2)) || [];
+
+                // Create a list of series based on the order of the fields,
+                // We need to flip it at the end to ensure the series stack right.
```

Comment:
Does the stacking matter that much as this is a line series and not an area series?